.github/workflows/build-images-ci.yaml

```diff
@@ -140,7 +143,7 @@ jobs:
 
       # main branch pushes
       - name: CI Build ${{ matrix.name }}
-        if: ${{ github.event_name != 'pull_request_target' && !startsWith(github.ref_name, 'ft/') }}
+        if: ${{ github.event_name != 'pull_request' && !startsWith(github.ref_name, 'ft/') }}
```

Comment:
Small nit, we could add the "platforms" as part of the matrix.include.

Reply:
done!